common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
What these icons doing on this branch?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104063420-92771280-5215-11eb-966b-74a2db671e2b.png)

</details>